beta/src/components/Search.tsx

```diff
@@ -95,13 +95,15 @@ export const Search: React.FC<SearchProps> = ({
       </Head>
 
       <button
+        aria-label="Search"
         type="button"
         className="inline-flex md:hidden items-center text-lg p-1 ml-4 lg:ml-6"
         onClick={onOpen}>
         <IconSearch className="align-middle" />
       </button>
 
       <button
+        aria-label="Search"
```

Comment:
This one shouldn't be necessary because there's text inside?

Reply:
I've removed it now. Thanks.